apps/api/src/server/utils/mapping/map-case-details.js

```diff
@@ -0,0 +1,68 @@
+import { pick } from 'lodash-es';
+import { mapSector } from '../../utils/mapping/map-sector.js';
+import { mapCaseStatus } from './map-case-status.js';
+import { mapDateStringToUnixTimestamp } from './map-date-string-to-unix-timestamp.js';
+import { mapGridReference } from './map-grid-reference.js';
+import { mapKeysUsingObject } from './map-keys-using-object.js';
+import { mapRegion } from './map-region.js';
+import { mapServiceCustomer } from './map-service-customer.js';
+import { mapValuesUsingObject } from './map-values-using-object.js';
+import { mapZoomLevel } from './map-zoom-level.js';
+
+/**
+ * @param {import('@pins/api').Schema.Case} caseDetails
+ * @returns {{
+ *  title?: string | undefined,
+ *  description?: string | undefined,
+ *  caseEmail?: string | undefined,
+ *  status?: string | undefined,
+ * 	gridReference?: { easting?: number | undefined, northing?: number | undefined },
+ *  regions: { displayNameEn?: string | undefined, displayNameCy?: string | undefined }
+ *  keyDates?: { firstNotifiedDate?: number | undefined, submissionDate?: number | undefined }
+ *  subSector?: { name: string | undefined, abbreviation: string | undefined, displayNameEn: string | undefined, displayNameCy: string | undefined },
+ *  sector?: { name: string | undefined, abbreviation: string | undefined, displayNameEn: string | undefined, displayNameCy: string | undefined },
+ *  applicant: { organisationName?: string | undefined, firstName?: string | undefined, middleName?: string | undefined, lastName?: string | undefined, email?: string | undefined, website?: string | undefined, phoneNumber?: string | undefined},
+ *  mapZoomLevel?: { displayNameEn: string | undefined, displayNameCy: string | undefined, locationDescription: string | undefined },
+ *  regionNames?: string[],
+ *  address?: { addressLine1?: string | undefined, addressLine2?: string | undefined, town?: string | undefined, county?: string | undefined, postcode?: string | undefined}}}
+ */
+export const mapCaseDetails = (caseDetails) => {
```

Comment:
Can we rename this to use the work `Application` instead of `Case` because we are returning information that specifically relates to applications, whereas cases are a more general catch-all term